ghost/admin/app/components/koenig-lexical-editor.js

```diff
@@ -669,34 +669,53 @@ export default class KoenigLexicalEditor extends Component {
         const multiplayerDocId = cardConfig.post.id;
         const multiplayerUsername = this.session.user.name;
 
+        const [hasInitialised, setHasInitialised] = React.useState(false);
+
+        const KGEditorComponent = ({isInitInstance}) => {
+            const handleInitInstance = (data) => {
+                if (this.args.initLexical && isInitInstance && !hasInitialised) {
+                    this.args.initLexical(data);
+                    setHasInitialised(true);
+                }
+            };
+            return (
+                <div style={isInitInstance && !hasInitialised ? {visibility: 'hidden', position: 'absolute'} : {}}>
+                    <KoenigComposer
+                        editorResource={this.editorResource}
+                        cardConfig={cardConfig}
+                        enableMultiplayer={enableMultiplayer}
+                        fileUploader={{useFileUpload, fileTypes}}
+                        initialEditorState={this.args.lexical}
+                        multiplayerUsername={multiplayerUsername}
+                        multiplayerDocId={multiplayerDocId}
+                        multiplayerEndpoint={multiplayerEndpoint}
+                        onError={this.onError}
+                        darkMode={this.feature.nightShift}
+                        isTKEnabled={true}
+                    >
+                        <KoenigEditor
+                            editorResource={this.editorResource}
+                            cursorDidExitAtTop={this.args.cursorDidExitAtTop}
+                            placeholderText={this.args.placeholder}
+                            darkMode={this.feature.nightShift}
+                            onChange={isInitInstance ? e => handleInitInstance(e) : this.args.onChange}
+                            registerAPI={this.args.registerAPI}
+                        />
+                        <WordCountPlugin editorResource={this.editorResource} onChange={this.args.updateWordCount} />
+                        <TKCountPlugin editorResource={this.editorResource} onChange={this.args.updatePostTkCount} />
```

Comment:
Same for both of these, we don't want the secondary instance interacting with any of the outside world aside from the main content onChange handler